tests/scripts/actionutils.sh

```diff
@@ -161,21 +173,60 @@ function check_client_configs() {
 }
 
 function bootstrap_head() {
-    # Bootstrap microceph on the head node
-    lxc exec node-head -- sh -c "microceph cluster bootstrap"
-    lxc exec node-head -- sh -c "microceph status"
+    set -x
+
+    arg=$1
```

Comment:
Nit: please use the `local` keyword for scope control
```
local arg="$1"
```

Reply:
ack